src/nectarchain/dqm/bokeh_app/app_hooks.py

```diff
@@ -13,7 +13,7 @@
                 'Results_MeanWaveForms_LowGain',
                 'Results_CameraMonitoring']
 
-geom = CameraGeometry.from_name("NectarCam-003")
+geom = CameraGeometry.from_name("NectarCam")
```

Comment:
I'm not sure this actually works as intended. I suspect it does not load a generic Geometry, but an older version preceding 003. I was rather suggesting to add a single parameter somewhere that can set the geometry all across the package.

Reply:
Perhaps it would be better, in this case, to have this line in a function that could be used everywhere.
Something like : 
```
def GetCameraGeometry(name="NectarCam-003"):  
      return CameraGeometry.from_name(name)
```

Reply:
Agree with @vmarandon , I was thinking about the same: maybe even easier to have a constant for the camera geometry version across the whole package, in something like `utils/constants.py`.

I will revert the last change and propose to address this in another PR, we planned to have one on re-organization of the code anyway.

Reply:
If you have a working EventSource, you can also take it's `SubarrayDescription` (`source.subarray`) and write it to an HDF5 file, and load that when needed. E.g.:
```py3
with EventSource("some_nectarcam_file.zfits.gz")  as source:
    source.subarray.to_hdf("subarray.h5")

# then later:

subarray = SubarrayDescription.from_hdf("subarray.h5")
geom = subarray.tel[1].camera.geometry
```

Reply:
Thanks a lot, @kosack .
Indeed, when working with an active EventSource, I agree using `source.subarray.tel[0].camera.geometry` is the most robust and self-consistent way to get the correct camera geometry.

In this PR, though, the context for the running Bokeh application is independent from any EventSource, the input data are fed from an object-oriented data base. They are stored as nested dictionaries there, and so we "lost" at that point part of the original meta-data. Just for visualization purpose, this was a way to correct the geometry right in camera displays.

Reply:
I guess it's only by luck though that the simulated pixel ids are the same as the real ones? That is not the case in general. So just beware if we switch simulations, it could "break" images in such a display.